dask/dataframe/core.py

```diff
@@ -2643,6 +2654,11 @@ def kurtosis(
            Further, this method currently does not support filtering out NaN
            values, which is again a difference to Pandas.
         """
+        if PANDAS_GT_200 and axis is None:
```

Comment:
Could be a decorator, since you do this in two places.

Reply:
Agreed. Though in this particular case I think the cognitive load on readers is actually smaller if we just duplicate this simple logic twice. If the logic was much more involved, or if it was used in lots of places, totally agree using a decorator (or some other isolated utility) would be a good practice.